### PR TITLE
docs(readme): update hacking documentation

### PR DESCRIPTION
**Pull request by bacongobbler**

### CODE REVIEW

**Comment by technosophos** [comment]
Yay helm stuff! Too bad we can't tell a better story than "log into a minion...". But that will come.

File: README.md [hunk 1]
```diff
@@ -2,63 +2,67 @@
...
 ```console
-$ make docker-build docker-push
+$ kubectl get pods --namespace=deis
 ```
 
-Finally create a PostgreSQL database and the Deis workflow service:
+Once this is done, you can SSH into the minion running the controller and run the following:
+
+```
+$ vagrant ssh minion-1
```
Comment (arschles) [inline README.md]: were there instructions on installing a kube cluster with vagrant?

fwiw I'm :+1: on using [micro-kube](https://github.com/krancour/micro-kube) if they're doing a local install.

Comment (bacongobbler) [inline README.md]: We could link to https://github.com/deis/workflow/blob/master/docs/src/installing-deis/quickstart.md#choose-a-provider, which is basically a proxy to kubernetes' documentation.

Comment (arschles) [inline README.md]: @bacongobbler SGTM

File: README.md [hunk 2]
```diff
@@ -2,63 +2,67 @@
...
+$ curl -sSL http://deis.io/deis-cli/install.sh | sh
+$ sudo mv deis /bin
+$ kubectl get service deis-workflow
+$ deis register 10.247.59.157 # or the appropriate CLUSTER_IP
```
Comment (bacongobbler) [inline README.md]: note that this relies on https://github.com/deis/charts/pull/20

**Comment by bacongobbler** [comment]
I’m gonna hold off on merging this until I've addressed @arschles's comment. Thanks for the reviews! :)

**Comment by bacongobbler** [comment]
Okay, I've added a link to the kubernetes "Getting Started" documentation. @arschles does this change LGTY?

**Comment by arschles** [comment]
@bacongobbler yep, still LGTM after changes. thanks

**Pull request merged**